examples/green_screen/MultiDeviceCapturer.h

```diff
@@ -20,7 +33,8 @@ class MultiDeviceCapturer
         bool master_found = false;
         if (device_indices.size() == 0)
         {
-            throw std::runtime_error("Capturer must be passed at least one camera!");
+            cerr << "Capturer must be passed at least one camera!\n ";
```

Comment:
For my own learning, why remove the throws and instead just `exit(1)`?

Reply:
For testing, main should return 0 for success. I was not sure if throws were randomly setting the return code so I set them explicitly. I was also not seeing the error message in the console output when throw was firing so the associated message was not helping.

Reply:
To add to the learning:

According to [cppreference](https://en.cppreference.com/w/cpp/error/terminate) when an exception is thrown and not caught, `std::terminate` is called. The default implementation of `std::terminate` is to call `std::abort` (though the terminate handler can be overridden at run time by the program). According to [cppreference](https://en.cppreference.com/w/cpp/utility/program/abort), `std::abort` causes abnormal program termination. Therefore, it should not set the exit code to 0. In fact, on Windows, `std::abort` results in an exit code of 3. I was unable to determine the exit code on Linux.

I'm fine with doing error handling this way. If we wanted to use the stack unwinding, we could also just catch `std::runtime_error` where ever the constructor for `MultiDeviceCapture` is.